org.beeref.BeeRef.json

```diff
@@ -0,0 +1,63 @@
+{
+    "id": "org.beeref.BeeRef",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "beeref",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--env=PATH=/usr/bin:/app/bin"
+    ],
+    "build-options": {
+        "env": {
+            "PYTHON": "/usr/bin/python3"
+        },
+        "build-args": [
+          "--share=network"
```

Comment:
this is not allowed, this will fail.

For python you need the add the dependencies to the manifest.
There are tool like that can help https://github.com/flatpak/flatpak-builder-tools